src/pages/networks/forms/NetworkForm.tsx

```diff
@@ -0,0 +1,732 @@
+                        formik.setFieldValue("fan_type", undefined);
+                        formik.setFieldValue("fan_overlay_subnet", undefined);
+                        formik.setFieldValue("fan_underlay_subnet", undefined);
+                        formik.setFieldValue("network", undefined);
+                        formik.setFieldValue("ipv4_l3only", undefined);
+                        formik.setFieldValue("ipv6_l3only", undefined);
+                      }
+                      if (e.target.value === "bridge-fan") {
+                        formik.setFieldValue("type", "bridge");
+                        formik.setFieldValue("bridge_mode", "fan");
+                        formik.setFieldValue("ipv4_address", undefined);
+                        formik.setFieldValue("ipv6_address", undefined);
+                        formik.setFieldValue("ipv6_nat", undefined);
+                        formik.setFieldValue("network", undefined);
+                        formik.setFieldValue("ipv4_l3only", undefined);
+                        formik.setFieldValue("ipv6_l3only", undefined);
+                      }
+                      if (e.target.value === "ovn") {
+                        formik.setFieldValue("type", "ovn");
+                        formik.setFieldValue("bridge_mode", undefined);
+                        formik.setFieldValue("fan_type", undefined);
+                        formik.setFieldValue("fan_overlay_subnet", undefined);
+                        formik.setFieldValue("fan_underlay_subnet", undefined);
+                        formik.setFieldValue("bridge_driver", undefined);
+                        formik.setFieldValue("dns_mode", undefined);
+                        formik.setFieldValue("ipv4_dhcp_expiry", undefined);
+                        formik.setFieldValue("ipv4_dhcp_ranges", undefined);
+                        formik.setFieldValue("ipv4_ovn_ranges", undefined);
+                        formik.setFieldValue("ipv6_dhcp_expiry", undefined);
+                        formik.setFieldValue("ipv6_dhcp_ranges", undefined);
+                        formik.setFieldValue("ipv6_ovn_ranges", undefined);
+                      }
+                    }}
+                    value={
+                      formik.values.type === "bridge"
+                        ? `bridge-${formik.values.bridge_mode ?? "standard"}`
+                        : formik.values.type
+                    }
+                    disabled={formik.values.readOnly}
+                  />
+                  {formik.values.type === "ovn" && (
+                    <Select
+                      {...getFormProps("network")}
+                      label="Uplink"
+                      help="Uplink network to use for external network access"
+                      options={getNetworkOptions()}
+                      required
+                      disabled={formik.values.readOnly}
+                    />
```

Comment:
It seems that with the CLI you can create an OVN network without explicitly passing the uplink parameter - see step 7 [here](https://documentation.ubuntu.com/lxd/en/latest/howto/network_ovn_setup/#set-up-a-lxd-cluster-on-ovn). It's probably picking some existing network by default.
I wonder if we should replicate the same approach in the UI, setting a default option for this dropdown, that the user can then change if they want, of course.

Reply:
Yes, the uplink is not a required parameter, I'll make it optional.